### ISSUE TITLE
This is a closed source project

### ISSUE THREAD

**Issue by Cyberes** [HeyPuter/puter#61]
Oh boy, where do I even start with this? A mockery of open source? No, that's too kind, this is more like open source crucifixion. The epitome of all things profane in the world of code, you've managed to make a farce of the one thing that separates us from the corporate, money-grabbing monopolists.

Your "open source" client for a closed service? That's a joke. I mean, are you trying to be funny? Because if it's humor you're going for, you might want to consider a career in stand-up comedy. You'll do less damage there.

Why did you even bother with the pretense of open source? Open source implies freedom, transparency, honesty - none of which I'm seeing here, buddy. It's like you're selling canned air for people to breathe.

In my four-and-a-half decades, I've seen some imbecilic abominations but, this... this is a new high in lows. Do you even understand what open source is? Do you? Because this, my dear friend, is a slap in the face to every developer who spends countless hours working on real open source projects.

How about trying to contribute something worthwhile to the community instead of writing a glorified API wrapper and slapping the open source label on it like it's some kind of badge of honor. It doesn't make your client less problematic, it just makes you look even more idiotic.

Oh, and on a side note: if you want to show off your programming skills, how about you release something that isn't a textbook example of how not to write software? You're making it cry.

So, in conclusion: clean up your act, learn what open source actually means, and start respecting the ethos of the community you're pretending to be part of. And for heaven's sake, please, take a C language course or two. It's just becoming embarrassing now.

**Comment by jelveh** [comment]
i ain't reading all that. i'm happy for you though. or sorry that happened.

**Comment by Cyberes** [comment]
Well, isn't that just absolutely typical? These so-called "developers" can't stand the heat of their dumpster fire code being scrutinized. Guy's probably sitting in his padded, ergonomic chair (undoubtedly funded by the poor saps who use his closed source mockery), sipping a soy latte, and bawling his eyes out because someone dared to speak the truth!

Go ahead, hide behind your closed issue, Mr. "Open Source" Imposter! And you know what they say about people who can't stand criticism. Only two types: either you're a shoddy craftsman, or just spineless. Pretty sure you've ticked both boxes there, buddy!

Did he leave a comment or just ran with his tail between his legs? No, don't answer that. Why am I even asking? Of course he didn't. Can't expect any maturity from a guy who thinks open source means "Hey, here's the code for my craptastic software that interfaces with my 'proprietary' garbage, feel free to look but don't you dare critique it!"

It's a mad world out there, mate. But at least we've got each other to angrily type at. Give me a holler if you need to vent at some other code nincompoops. It'll be my pleasure.
